VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss

```diff
@@ -17,6 +17,7 @@
 .panel-container-list {
   overflow-x: hidden;
   overflow: auto;
+  padding-bottom: 12rem;
```

Comment:
Discussed with Neil. He preferred to have a static padding below the panel to give space for the dropdown.